content/developer/misc/i18n/localization.rst

```diff
@@ -303,123 +588,68 @@ To create a new fiscal position, simply use the ``account.fiscal.position.templa
-        By default, the sequence is 10. -->
-        <field name="sequence" eval="..."/>
 
-        <!-- [Optional] Define an account. -->
-        <field name="account_id" ref="..."/>
-
-        <!-- [Optional] Define a label to be added to the journal item. -->
-        <field name="label">...</field>
+.. code-block:: py
 
-        <!-- [Optional] Define the type of amount_type, either 'fixed' or 'percentage'.
-        The last one is the default value. -->
-        <field name="amount_type">...</field>
+    {
+        "name": "COUNTRY - Accounting Reports",
+        "category": "Accounting/Localizations/Reporting",
+        "version": "1.0.0",
+        "license": "OEEL-1",
+        "depends": [
+            "l10n_XX", "account_reports"
+        ],
+        "data": [
+            "data/account_financial_html_report_data.xml",
+        ],
+        "auto_install": True,
+    }
 
-        <!-- [Optional] Define the balance amount on which this model will be applied to (100 by default).
-        Fixed amount will count as a debit if it is negative, as a credit if it is positive. -->
-        <field name="amount">...</field>
 
-        <!-- [Optional] Define eventually a tax. -->
-        <field name="tax_id" ref="..."/>
+Functional overview of financial reports is here: :doc:`/applications/finance/accounting/reporting/overview/main_reports`.
 
-        <!-- [Optional] The sames fields are available twice.
-        To enable a second journal line, you can set this field to true and
-        fill the fields accordingly. -->
-        <field name="has_second_line" eval="..."/>
-        <field name="second_account_id" ref="..."/>
-        <field name="second_label">...</field>
-        <field name="second_amount_type">...</field>
-        <field name="second_amount">...</field>
-        <field name="second_tax_id" ref="..."/>
-   </record>
+Some good examples:
 
-How to create a new dynamic report?
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+* `l10n_ch_reports/data/account_financial_html_report_data.xml <https://github.com/odoo/enterprise/blob/15.0/l10n_ch_reports/data/account_financial_html_report_data.xml>`__
+* `l10n_be_reports/data/account_financial_html_report_data.xml <https://github.com/odoo/enterprise/blob/15.0/l10n_be_reports/data/account_financial_html_report_data.xml>`__
 
-If you need to add some reports on your localization, you need to create a new module named **l10n_xx_reports**.
-Furthermore, this additional module must be present in the ``enterprise`` repository and must have at least two dependencies,
-one to bring all the stuff for your localization module and one more, ``account_reports``, to design dynamic reports.
+For the fields meaning dive directly to the source:
 
-.. code-block:: py
+* `account.financial.html.report (v15) <https://github.com/odoo/enterprise/blob/d4eff9d39469cf3fe18589a1547cb0cdb93f4ae9/account_reports/models/account_financial_report.py#L59-L75>`__
```

Comment:
In financial reports, we have the special date changer as well. 

It would not be bad to have some section about how the P&L and balance sheet can be linked @Shawcker and how the current year earnings (and previous year earnings), ... work @Levizar

Reply:
Maybe improve it later? This PR is already big enough

Reply:
I don't think we should explain financial reports here. As stated in my previous comments, this is not the aim of this page, and if we do, it'll become a messy box were we put everything. We should have dedicated pages for the different report engines that are available for XML definition, and only link those pages here.